tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp

```diff
@@ -58,10 +58,10 @@ struct not_const_formattable_type {
 template <>
 struct std::formatter<basic_custom_formattable_type, char> {
     constexpr basic_format_parse_context<char>::iterator parse(basic_format_parse_context<char>& parse_ctx) {
-        if (parse_ctx.begin() != parse_ctx.end()) {
+        if (parse_ctx.begin() != parse_ctx.end() && *parse_ctx.begin() != '}') {
             throw format_error{"only empty specs please"};
         }
-        return parse_ctx.end();
+        return parse_ctx.begin();

```

Comment:
This should be non-functional, correct? It's a matter of if we "match" the closing '}' or not.